mexico-city/Cargo.toml

```diff
@@ -32,6 +32,7 @@ wasmi = { git = "https://github.com/veracruz-project/wasmi.git", branch = "verac
 hex = { version = "0.4.1", default-features = false }
 serde_json = { git = "https://github.com/veracruz-project/json.git", branch = "veracruz" }
 err-derive = "0.2"
+pinecone = "*"
```

Comment:
We probably want a fixed version of `pinecone` here, rather than a wildcard.  Also, what is `pinecone` being used for in Mexico City?

Reply:
This will be removed once I change the new host calls to match C-style function call.